srcpkgs/blockify/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'blockify'
+pkgname=blockify
+version=3.6.3
+revision=1
+archs="x86_64"
```

Comment:
Why restrict the archs? Something go wrong with crossbuild or even x86_64-musl?

Reply:
well, spotify only exists for x86_64 right now, so it is more of a redundancy really. I could remove it so if spotify becomes available in the future on these archs too i won't need to change it.

Reply:
Ah, I see. Nothing with this, but the spotify app that it depends on. I might add a comment explaining. `archs="x86_64" # spotify only on x86_64 arch`

Reply:
okey, will do.
What to do with spotify tho. I could remove it from depends, because it is in restricted category, and just open it as issue for `spotify` package